packages/edit-navigation/src/components/secondary-sidebar/inserter-sidebar.js

```diff
@@ -0,0 +1,89 @@
+/**
```

Comment:
My personal preference would be for the path to this file to be `packages/edit-navigation/src/components/inserter-sidebar/index.js`, as I'd find that easier to locate.

Reasoning - Secondary sidebar is where the inserter sidebar is rendered, but similar to the classname issue, if it were ever rendered anywhere else the folder name would have to be changed.

Reply:
Good point. I was feeling a little uneasy about this. Your comment has made my concerns clearer.

Reply:
✅